input/fsh/SD_PMSIGHM.fsh

```diff
@@ -0,0 +1,32 @@
+Profile:        PMSIGHM
+Parent:         Observation
+Id:             pmsi-grouped-homogene-maladesd-dpdr
+Title:          "Groupes homogènes de Malades"
+Description:    "Profil pour la ressource Groupes Homogènes de Malades du PMSI"
```

Comment:
I don't think you have to write "Profil pour la ressource", it doesn't give any information
Maybe something like "Observation sur le Groupe Homogène de Malade (GHM) d'un patient"

Reply:
👍